freshmaker/handlers/__init__.py

```diff
@@ -611,6 +611,9 @@ def build_image(build):
                 # We skip this image for now. It will be built once the ODCS
                 # compose is finished.
                 return
+            except Exception as ex:
+                self.log_warn("While processing the event with id {}, exception"
```

Comment:
Could you make this use `log_exception` so that we get the whole traceback?

Reply:
Are you sure that it's ok to use flask's `log_exception`? I think it exists for more internal things, and there should be an active HTTP request. Maybe it's better to just use `traceback.print_exc` to print exception info with the traceback?

Reply:
@apaplaus I meant just creating a wrapper like `self.log_warn` but for exceptions. It wouldn't be using Flask in this case since this is the backend code.

Reply:
Ohhh, ok) I wondered what is that mysterious method that I don't know about, so I found one in Flask)